libsolidity/ast/AST.cpp

```diff
@@ -202,36 +208,6 @@ vector<pair<util::FixedHash<4>, FunctionTypePointer>> const& ContractDefinition:
 	return *m_interfaceFunctionList;
 }
 
-vector<Declaration const*> const& ContractDefinition::inheritableMembers() const
```

Comment:
removed because unused?

Reply:
Yes, it was only used in ``TypeType::nativeMembers`` and that now uses the new ``declarations()`` instead.